README.md

```diff
@@ -1,42 +1,42 @@
 # Hex Monscape 👾
 
-[![Build & Deploy](https://github.com/Haraj-backend/hex-monscape/actions/workflows/deploy.yml/badge.svg)](https://github.com/Haraj-backend/hex-monscape/actions/workflows/deploy.yml) [![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT) [![Go Report Card](https://goreportcard.com/badge/github.com/Haraj-backend/hex-monscape)](https://goreportcard.com/report/github.com/Haraj-backend/hex-monscape) 
+[![Go Reference](https://pkg.go.dev/badge/github.com/Haraj-backend/hex-monscape.svg)](https://pkg.go.dev/github.com/Haraj-backend/hex-monscape)
+[![Go Report Card](https://goreportcard.com/badge/github.com/Haraj-backend/hex-monscape)](https://goreportcard.com/report/github.com/Haraj-backend/hex-monscape)
+[![coverage](https://img.shields.io/badge/coverage-82.2%25-brightgreen)](./.output/cov-report/coverage.out)
 
-Welcome to this repo!
+`Hex-Monscape` is a simple turn-based game created for Haraj Solutions Team onboarding process.
 
-In this repo you will learn about Haraj Solutions Team's secret technique in writing maintainable code for Haraj projects. 🥷🏻💥
+Inside the game source code, you can find the team's secret technique in writing production-grade code for Haraj web services that help serve millions of users daily in Saudi Arabia, despite only having a handful of team members. 🥷🏻💥
```

Comment:
Minor suggestion here, how if we put Haraj URL (haraj.com.sa) in `Haraj` word in this sentence?

"...team's secret technique in writing production-grade code for **[Haraj](https://haraj.com.sa)** web services..."